Styles/style.css

```diff
@@ -130,26 +144,27 @@ body {
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 2.5rem;
-    height: 2.3rem;
+    width: 3rem;
+    height: 3rem;
     border-radius: 50%;
     bottom: 1rem;
-    right: 2.1rem;
+    right: 3rem;
     z-index: 1;
     border: none;
     cursor: pointer;
     outline: none;
     background: #2d323b;
-    box-shadow: 4px 6px 3px #292e36, -4px -3px 4px #313640;
-    /* background-color: transparent; */
-    /* box-shadow: 0 5px 10px rgba(0, 0, 0, 0.2); */
+    /* box-shadow: 4px 6px 3px #292e36, -4px -3px 4px #313640; */
   }
   .btn-toggle:active {
     box-shadow: 0 5px 10px rgba(0, 0, 0, 0.2);
   }
+  .social-links {
+    right: -1rem;
+  }
   .btn-toggle .img-btn {
-    width: 2rem;
-    height: 2rem;
+    width: 2.5rem;
+    height: 2.5rem;
     -webkit-animation: spin 4s linear infinite;
     -moz-animation: spin 4s linear infinite;
     animation: spin 4s linear infinite;
```

Comment:
In my opinion, having a spinner toggle button that spins infinitely is a bad UX. It distracts the user from the actual content, and doesn't contribute in making the UI attractive. Instead try to have a different home page and put the spinner there as a logo. But again. that's subjective, if you like it, let it be.